crates/env/src/engine/off_chain/test_api.rs

```diff
@@ -306,7 +306,8 @@ where
         instance.engine.set_callee(encoded_alice.clone());
 
         // set up the funds for the default accounts
-        let substantial = 1_000_000;
+        // the 1_000_000_000 is the same value as in the e2e tests
+        let substantial = 1_000_000_000;
```

Comment:
I think the original `1_000_000` value was somewhat arbitrary anyway, so makes sense to change it to be the same as the `substrate-contracts-node` used for testsin.

Is it only the Alice account that is endowed with this amount? I see bob and charlie are still set to `1_000` below - does that match up with the endowed amounts for the e2e tests too? Seems kind of low.

Reply:
That's true, i will check the behaviour and make it equal for all name accounts (at least). The accounts created with `::From([1;32])` should have a balance of 0 or the existential minimum?